testing-common/src/test/groovy/context/FieldBackedProviderTest.groovy

```diff
@@ -5,11 +5,11 @@
 
 package context
 
-import static context.ContextTestInstrumentation.IncorrectCallUsageKeyClass
-import static context.ContextTestInstrumentation.IncorrectContextClassUsageKeyClass
-import static context.ContextTestInstrumentation.IncorrectKeyClassUsageKeyClass
-import static context.ContextTestInstrumentation.KeyClass
-import static context.ContextTestInstrumentation.UntransformableKeyClass
+import static ContextTestInstrumentationModule.IncorrectCallUsageKeyClass
+import static ContextTestInstrumentationModule.IncorrectContextClassUsageKeyClass
+import static ContextTestInstrumentationModule.IncorrectKeyClassUsageKeyClass
+import static ContextTestInstrumentationModule.KeyClass
+import static ContextTestInstrumentationModule.UntransformableKeyClass
```

Comment:
this class doesn't look like it changed package?

Reply:
😮 
That surprised me, I don't know why would IntelliJ do this - reverted.